hack/windows/Helper.psm1

```diff
@@ -1,9 +1,10 @@
 function Get-WebFileIfNotExist($Path, $URL) {
-    if (Test-Path $Path) {
-        return
+    $count=0
+    while (!(Test-Path $Path) -and ($count -ne 3)) {
+        $count++
+        Write-Host "Downloading $URL to $PATH"
+        curl.exe -skLo $Path $URL
```

Comment:
I think we shouldn't disable server certificate verification, especially when the script is also used by users.